lib/features/composer/presentation/model/inline_image.dart

```diff
@@ -1,25 +1,19 @@
 
 import 'package:equatable/equatable.dart';
-import 'package:model/model.dart';
-import 'package:tmail_ui_user/features/composer/presentation/model/image_source.dart';
+import 'package:model/upload/file_info.dart';
 
 class InlineImage with EquatableMixin {
-  final ImageSource source;
-  final String? link;
-  final FileInfo? fileInfo;
-  final String? cid;
```

Comment:
why we remove these?

Reply:
It is no longer in use.

Reply:
how about if it have `link`?

Reply:
Currently we do not support inserting images via `link`. IMO, we should remove unnecessary variables to avoid confusion for developers later. Furthermore, inserting images via links is quite easy, we already have support functions available.